Formula/nef.rb

```diff
@@ -1,16 +1,10 @@
 class Nef < Formula
   desc "💊 steroids for Xcode Playgrounds"
   homepage "https://nef.bow-swift.io"
-  url "https://github.com/bow-swift/nef/archive/0.6.0.tar.gz"
-  sha256 "a67bb4201739898832ec52e92d838ccfcae25a490df433cd7cf44f1fc6e0a786"
+  url "https://github.com/bow-swift/nef/archive/0.6.1.tar.gz"
+  sha256 "182d5f91e3048f046a63bf81d148f94acc7c667b5314cf76f36a9fb75efdb3a7"
 
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "4dffdd30d80ab45cda870e0e5ebe981f64ab878de8cabaeb13bb33cc5ef6d1a0" => :catalina
-    sha256 "49c50ed996365fa2636500525c5ab74554eb530afde00e94aaf004c2032fcf57" => :mojave
-  end
-
```

Comment:
You don't need to touch the bottle block. This is automatically managed when merging.